keyboards/owlab/jelly/info.json

```diff
@@ -0,0 +1,95 @@
+{
+    "keyboard_name": "jelly",
+    "url": "https://www.popkeyboard.com/products/jrrgvdsyu47wer",
+    "maintainer": "Owlab",
+    "width": 16,
+    "height": 6.25,
+    "layouts": {
+        "LAYOUT": {
```

Comment:
This does not match the content of jelly.h

Reply:
This is actually match the content of Jelly.h. In order to reduce pins in wiring, Delete, Home and other keys are arranged on other rows. Attached here is the physical layout picture and the circuit schematic diagram for your reference.
[https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/e8661ef182b17d75892f6664ea3581d9.jpg?v=1618594274](url)
[https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/Jelly_hotswap_circuit_schematic.png?v=1629449798](url)

Reply:
https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/e8661ef182b17d75892f6664ea3581d9.jpg?v=1618594274
https://cdn.shopifycdn.net/s/files/1/0508/8716/4061/files/Jelly_hotswap_circuit_schematic.png?v=1629449798

Reply:
https://www.reddit.com/r/confidentlyincorrect -  LAYOUT doesnt exist for this keyboard